examples/mean/tests/test_mean.py

```diff
@@ -99,30 +103,33 @@ def mean_randomised_test(dut):
     scoreboard = Scoreboard(dut)
     scoreboard.add_interface(dut_out, exp_out)
 
-    data_width = int(dut.DATA_WIDTH.value)
-    bus_width = int(dut.BUS_WIDTH.value)
+    DATA_WIDTH = int(dut.DATA_WIDTH.value)
+    BUS_WIDTH = int(dut.BUS_WIDTH.value)
     dut._log.info('Detected DATA_WIDTH = %d, BUS_WIDTH = %d' %
-                  (data_width, bus_width))
+                  (DATA_WIDTH, BUS_WIDTH))
 
     cocotb.fork(clock_gen(dut.clk, period=CLK_PERIOD_NS, units='ns'))
```

Comment:
This should be replaced with `Clock` as well.

Reply:
Done